src/lints/union_must_use_removed.ron

```diff
@@ -0,0 +1,68 @@
+SemverQuery(
+    id: "union_must_use_removed",
+    human_readable_name: "union #[must_use] removed",
+    description: "A union is no longer marked with #[must_use].",
+    required_update: Minor,
+    lint_level: Allow,
+
```

Comment:
```suggestion
```
Since we don't have auto-formatting at the moment, please manually ensure the files are appropriately formatted and don't have unexpected whitespace.